src/vs/editor/standalone/browser/quickInput/standaloneQuickInputService.ts

```diff
@@ -96,6 +97,9 @@ export class StandaloneQuickInputService implements IQuickInputService {
 		@ICodeEditorService private readonly codeEditorService: ICodeEditorService
 	) {
 	}
+	formatPinnedItems(storageKey: string, quickPick: IQuickPick<IQuickPickItem>, storageService: IStorageService, callback?: () => Promise<void>): void {
+		return this.activeService.formatPinnedItems(storageKey, quickPick, storageService, callback);
+	}
```

Comment:
I think Tyler wanted this just as a util function, outside of the QuickInputService(s)